packages/account-tree-controller/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Bump `@metamask/utils` from `^11.8.0` to `^11.8.1` ([#6708](https://github.com/MetaMask/core/pull/6708))
```

Comment:
The changelog checker is rightly complaining because this should be listed under Unreleased, can you move it there?